games-arcade/ccatch/ccatch-5.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Help a cat catch birds"
+DESCRIPTION="cCATch is a portation of the game CATch, written in c using SDL"
+HOMEPAGE="https://www.github.com/TheZeldakatze/c-CATch"
+COPYRIGHT="2020 Victor Gamper"
+LICENSE="GNU GPL v3"
+REVISION="2"
+srcGitRev="018c0fd7184e02992488a95ce514c3f1d77d7604" 
+# 6 Dec 2021 commit.
+SOURCE_URI="https://github.com/TheZeldakatze/c-CATch/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="c5f0d7b696d641b6168b92b05e822f76c92de31c784739d94f901b5596e593c5"
+SOURCE_FILENAME="c-CATch-$srcGitRev.tar.gz"
+SOURCE_DIR="c-CATch-$srcGitRev"
+ADDITIONAL_FILES="ccatch.rdef"
+
+ARCHITECTURES="all"
+
+PROVIDES="
+	cCATch = $portVersion
+	app:cCATch = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libSDL_1.2$secondaryArchSuffix
```

Comment:
No need for `$secondaryArchSuffix` if this builds with `gcc2` (further down also), if not it should be added to other places as well and `x86_gcc2` should be marked broken (`!x86_gcc2`) in `ARCHITECTURES`.

Reply:
I tried to compile it using x86_gcc2 in a virtual machine. It didn't compile as it is using c99.
So I changed the ARCHITECTURES to:
`ARCHITECTURES="all !x86_gcc2"
SECONDARY_ARCHITECTURES="x86"`

How do I test the x86_32 variant?

Reply:
You build it with `hp ccatch_x86` then ...

Reply:
Did a quick check, doesn't build for gcc2 (parse errors), so best to stick with `SECONDARY_ARCHITECTURES="x86"` then :)